CONTRIBUTING.md

```diff
@@ -34,7 +34,9 @@ Once you've worked out details with the maintainers, follow these general steps:
 
 1. Make your changes to the source files
     * For notebook-based tutorials, we recommend making changes to the `.py` version
-    then syncing changes with `tox -e my_tutorial_dir -- sync`.
+    then syncing changes with `tox -e my_tutorial_dir -- sync`. Alternatively, if
+    you have already run all the cells in your browser, you can select
+    `File` &rarr; `Jupytext` &rarr; `Pair Notebook with Percept Script` to save the ipynb.
```

Comment:
nit: typo Percept

unclear. "to save the outputs directly to the notebook version"